client/blocks/taxonomy-manager/list-item.jsx

```diff
@@ -156,9 +157,11 @@ class TaxonomyManagerListItem extends Component {
 							{ translate( 'Delete' ) }
 						</PopoverMenuItem>
 					}
-					<PopoverMenuItem href={ this.getTaxonomyLink() } icon="external">
-						{ translate( 'View Posts' ) }
-					</PopoverMenuItem>
+					{ ! isJetpack &&
+						<PopoverMenuItem href={ this.getTaxonomyLink() } icon="external">
```

Comment:
Well! this is not directly related but I think we could move the logic contained in `getTaxonomyLink` to `state/sites/selectors` and we can test it properly. This function could return a falsy value for Jetpack sites. The component here won't have any specific JetPack logic but just checks the falsy value instead. Thoughts on this approach ?